### PR TITLE
Fix links in CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by houhoulis**
I am providing these changes to make it easier for a new contributor (like me) to find necessary information.

I have questions about 3 links in this file:

1. **"The Website" in the top section:** The Contributing guide doesn't exist, and the repo (https://github.com/exercism/website) is not accepting contributions. I linked to the README in case the website opens up in the future. Was that the right choice?
2. **"The Exercises API" in the top section:** I found a link that resolves, but is the API not a thing? Should I remove this line?
3. **"Test Data Versioning":** There is text in `CONTRIBUTING.md` that is not on https://github.com/exercism/problem-specifications/blob/main/CONTRIBUTING.md about test data versioning. It looks like that section is also in `README.md`, but not on https://github.com/exercism/problem-specifications/blob/main/README.md. Perhaps it's obsolete. I'm not sure how it can be in `main` but not display on GitHub dot com, so I don't know how or whether to fix that link. (It would be nice for these files and GitHub dot com to be consistent, for a newcomer.) The test data versioning link, which I didn't touch, is on line 192-193 in this PR. Should I change that link or delete those two sentences?

I am more than happy for any feedback or corrections, as I would like to use this guide to help me make a PR to add some tests to an exercise. Thanks.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ We'll do everything we can to help you get started.
 The two best ways to get help are to
 
 * jump into the [support chat](https://gitter.im/exercism/support).
-* open a [GitHub issue](https://github.com/exercism/exercism.io/issues).
+* open a [GitHub issue](https://github.com/exercism/problem-specifications/issues).
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:54]: I think you might have missed this bit at the top of this file:

> This guide covers several common scenarios pertaining to improving the language tracks themselves. There are other guides about contributing to other parts of the Exercism ecosystem. As such, the links are not specific to problem-specifications.

```suggestion
* open a [GitHub issue](https://github.com/exercism/exercism/issues).
```

I do agree that this is weird, and we'll likely move much of this content to our docs at some point.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ email Katrina at kytrinyx@exercism.io.
 ## Code of Conduct
 
 Help us keep Exercism welcoming. Please read and abide by the
-[Code of Conduct](https://github.com/exercism/exercism.io/blob/master/CODE_OF_CONDUCT.md).
+[Code of Conduct](https://github.com/exercism/problem-specifications/blob/main/CODE_OF_CONDUCT.md).
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:67]:
```suggestion
[Code of Conduct](https://exercism.org/docs/using/legal/code-of-conduct).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ We are grateful for any help in making Exercism better!
 This guide covers several common scenarios pertaining to **improving the language tracks**
 themselves. There are other guides about contributing to other parts of the Exercism ecosystem.
 
-* [The Command-Line Client](https://github.com/exercism/cli/blob/master/README.md)
-* [The Website](https://github.com/exercism/exercism.io/blob/master/CONTRIBUTING.md)
-* [The Exercises API](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md) (used by both the command-line client and the website)
+* [The Command-Line Client](https://github.com/exercism/cli/blob/main/README.md)
+* [The Website](https://github.com/exercism/website/blob/main/README.md)
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:20]: We currently don't plan on allow contributions to the website, so maybe remove this link for now?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,9 +16,9 @@ We are grateful for any help in making Exercism better!
...
+* [The Exercises API](https://github.com/exercism/DEPRECATED.x-api/blob/master/CONTRIBUTING.md) (used by both the command-line client and the website)
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:21]: I think we should also remove this link, as the information seems to be quite outdated. Agree @iHiD?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -121,8 +121,9 @@ Some of the problems also have a JSON file containing canonical test cases. Thes
 are used to hand-craft a test suite generator, allowing us to quickly regenerate
 test suites when edge cases or errors are discovered.
 
-The generic problem descriptions live in the `exercises` folder in this repository
-([exercism/problem-specifications](https://github.com/exercism/problem-specifications)).
+The generic problem descriptions live in the `exercises`
+[folder](https://github.com/exercism/problem-specifications/tree/main/exercises)
+in this repository ([exercism/problem-specifications](https://github.com/exercism/problem-specifications)).
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:126]:
```suggestion
in [this repository](https://github.com/exercism/problem-specifications).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -233,7 +234,7 @@ A problem must have a unique slug. This slug is used as
 ### In exercism/<TRACK_ID>
 
 * Do the same as when [porting an
-  exercise](#porting-an-exercise-to-another-language-track).
+  exercise](https://github.com/exercism/legacy-docs/blob/main/you-can-help/implement-an-exercise-from-specification.md).
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:237]: This link also points to a document with outdated information. It's probably a little bit better than nothing, but we should really be adding this information to our docs website :)

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -317,7 +318,8 @@ scenarios in this guide.
 
 ### Pull Request Guidelines
 
-See the [pull request guidelines](https://github.com/exercism/docs/blob/master/contributing/pull-request-guidelines.md) in the docs repository.
+See the [pull request guidelines](https://github.com/exercism/legacy-docs/blob/main/contributing/pull-request-guidelines.md)
+in the [legacy-docs](https://github.com/exercism/legacy-docs) repository.
 
 ### Exercise Versioning
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:324]: We no longer have versioning for exercise, so this section can be removed (I'd do that in a separate PR though).

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -339,15 +341,16 @@ help avoid unnecessary discussions about failing the current test suites.
...
+both the test suite and the reference solution.
 
-Please review the details in [README.md](https://github.com/exercism/problem-specifications#test-data-versioning) and bump the version number accordingly.
+Please review the details in the [README.md](https://github.com/exercism/problem-specifications#changing-tests).
 
 ### Anatomy of an Exercise
 
-See the [anatomy of an exercise](https://github.com/exercism/docs/tree/master/language-tracks/exercises/anatomy) in the docs repository.
+See the [anatomy of an exercise](https://github.com/exercism/legacy-docs/tree/main/language-tracks/exercises/anatomy),
+including reference solutions, in the [legacy-docs](https://github.com/exercism/legacy-docs) repository.
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:353]:
```suggestion
See the [Practice Exercises](https://exercism.org/docs/building/tracks/practice-exercises) page for more information on how to structure an exercise.
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -375,8 +378,15 @@ Optional keys:
 * `ignore_pattern` - A (case insensitive) regex pattern that will cause files matching it to not be served to the student by `exercism fetch`. The default value used if this key is not present is `example`
 * `solution_pattern` - A (case sensitive) regex pattern that matches solution files in the track repository. Used by [configlet](https://github.com/exercism/configlet) to check for the presence of an example solution for each problem implemented by the track.  The default value used if this key is not present is `[Ee]xample`.
 
+### Track-level Linting with Configlet
+
+For more details, see [this guide](https://github.com/exercism/legacy-docs/blob/main/language-tracks/configuration/configlet.md)
+in the [legacy-docs](https://github.com/exercism/legacy-docs) repository.
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:384]:
```suggestion
For more details on how to have the [configlet tool](https://exercism.org/docs/building/configlet) check your track via its linting functionality, see [this guide](https://exercism.org/docs/building/configlet/lint).
```

**Review by ErikSchierboom — COMMENTED** [review]
Thanks for working on this! In short: we're behind on porting our documentation. I expect most of the files that are linked to be integrated in the [docs part of the website](https://exercism.org/docs/). For now, we should link to new documentation when it exists, remove links that are not useful at the moment, and continue linking to old documents when it makes sense.

I've left some comments.

**Comment by houhoulis** [comment]
@ErikSchierboom I'm happy to make all these changes. Should I wait for @iHiD to answer your question? (https://github.com/exercism/problem-specifications/pull/1841#discussion_r712810675)

I'll have to rebase as well, since there seem to be conflicts.

File: CONTRIBUTING.md (hunk 4 above)
Comment (iHiD) [reply CONTRIBUTING.md:21]: Agreed

**Comment by ErikSchierboom** [comment]
@houhoulis There was an answer by @iHiD, so I think you can update now.

**Comment by ErikSchierboom** [comment]
@houhoulis Are you still interested in working on this PR?

**Comment by houhoulis** [comment]
Sorry @ErikSchierboom , and thanks for the opportunity/reminder to make the necessary changes. I will do so this weekend.

**Comment by ErikSchierboom** [comment]
Great!

**Comment by houhoulis** [comment]
@ErikSchierboom I believe I addressed all of your feedback -- as of September. Sorry for the delay. Thanks!

**Comment by wolf99** [comment]
Ah, This PR may need to be adjusted with respect to #1895 , or vice-versa, depending on peoples preference ?

**Comment by SleeplessByte** [comment]
@wolf99 would you be okay to update that after we merge this, given that this is content fixes and that is formatting?

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -116,8 +116,9 @@ Some of the problems also have a JSON file containing canonical test cases. Thes
 are used to hand-craft a test suite generator, allowing us to quickly regenerate
 test suites when edge cases or errors are discovered.
 
-The generic problem descriptions live in the `exercises` folder in this repository
-([exercism/problem-specifications](https://github.com/exercism/problem-specifications)).
+The generic problem descriptions live in the `exercises`
+[folder](https://github.com/exercism/problem-specifications/tree/main/exercises)
+in [this repository](https://github.com/exercism/problem-specifications).
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:121]: I think maybe be specific here?

```suggestion
in [the exercism/problem-specifications repository](https://github.com/exercism/problem-specifications).
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -311,20 +312,26 @@ scenarios in this guide.
...
-See the [anatomy of an exercise](https://github.com/exercism/docs/tree/main/language-tracks/exercises/anatomy) in the docs repository.
+See the [Practice Exercises](https://exercism.org/docs/building/tracks/practice-exercises) page for more information on how to structure an exercise.
 
 ### Track configuration file
 
 Each language track has a configuration file called `config.json`. Its structure is defined in [this spec](https://github.com/exercism/docs/blob/main/anatomy/tracks/config-json.md). The `config.json` file's key feature is that it defines which exercises the language track has implemented.
 
 The [configlet](https://github.com/exercism/docs/blob/main/anatomy/tracks/configlet/README.md) tool can verify (lint) a track's configuration file.
 
+### Track-level Linting with Configlet
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:327]:
```suggestion
### Track-level Linting with configlet
```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -311,20 +312,26 @@ scenarios in this guide.
...
+
+For more details on how to have the [configlet tool](https://exercism.org/docs/building/configlet) check your track via its linting functionality, see [this guide](https://exercism.org/docs/building/configlet/lint).
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:329]:
```suggestion
For more details on how to have the [configlet tool](https://exercism.org/docs/building/configlet) verify your track via its linting functionality, see [this guide](https://exercism.org/docs/building/configlet/lint).
```

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -311,20 +312,26 @@ scenarios in this guide.
...
+
 ### Git Basics
-See [Git Basics](https://github.com/exercism/docs/blob/main/contributing/git-basics.md) in [docs](https://github.com/exercism/docs) repository.
+
+See [Git Basics](https://github.com/exercism/legacy-docs/blob/main/contributing/git-basics.md)
```
Comment (ErikSchierboom) [inline CONTRIBUTING.md:333]:
```suggestion
For help with Git, see [Git Basics](https://github.com/exercism/legacy-docs/blob/main/contributing/git-basics.md)
```

**Review by ErikSchierboom — APPROVED** [review]
Just a couple of minor tweaks. I'm approving in advance.

**Comment by houhoulis** [comment]
@ErikSchierboom Done. Thanks for the feedback.

**Review by ErikSchierboom — APPROVED** [review]

**Review by kotp — APPROVED** [review]
Looks good.  Thanks for the effort!

**Review by SleeplessByte — APPROVED** [review]
